closure/compiler/test/goog_es6_interop/string.js

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 The Closure Rules Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview A goog.provide file for testing interoperability.
+ */
+goog.provide('person.string');
+
+/**
+ * @param {string} str String value to capitalize.
+ * @return {string} String value with first letter in uppercase.
+ */
+person.string.capitalize = function(str) {
```

Comment:
Could we simply add this method method to person.js instead?

Reply:
The test is for testing imports across es6 modules and goog.provide.

Reply:
Ah I missed that point =)